components/NewsletterCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+type NewsletterProps = {
+  url: string;
+  title: string;
+  date: string;
```

Comment:
Hello @Akshaybagai52,  
Here how's about using `Date` data type instead of `string`.

Reply:
Yes I will fix this